internal/provider/service_account_resource.go

```diff
@@ -86,7 +86,7 @@ func (r *serviceAccountResource) Metadata(_ context.Context, req resource.Metada
 
 func (r *serviceAccountResource) Schema(ctx context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		Description: "Provisions a Temporal Cloud Service Account.",
+		Description: "Provisions a Temporal Cloud Service Account. To avoid overwriting service account permissions, manage a specific service account in one and only one .tf file. ",
```

Comment:
I'm not sure this is necessary to say here, it seems as though this is an invalid way to use Terraform altogether

Reply:
TY for the input. The reason this was provided is that users were attempting to do this, which led to the underlying issue - #119 

IIRC, we agreed to update our docs and the TF docs to make this recommendation explicitly